hera_pspec/pspecdata.py

```diff
@@ -662,6 +666,16 @@ def C_model(self, key, model='empirical', time_index=None, known_cov=None):
             # add model to key
             Ckey = ((dset, dset), (bl,bl), ) + (model, time_index, False, True,)
 
+        elif model == 'outer_product':
```

Comment:
It seems like the last three conditionals can be merged into a single statement: `elif model in ['autos', 'outer_product', 'foreground_dependent']:` unless you suspect that in the future they may break apart in their Ckey definition

Reply:
Sure. Nice suggestions!